client/orm/structs/clauses/order.go

```diff
@@ -0,0 +1,100 @@
+package clauses
+
+import "strings"
+
+type Sort int8
+
+const (
+	SortNone       Sort = 0
+	SortAscending  Sort = 1
+	SortDescending Sort = 2
+)
+
+type OrderOption func(order *Order)
+
+type Order struct {
+	column string
+	sort   Sort
+	isRaw  bool
+}
+
+func OrderClause(options ...OrderOption) *Order {
+	o := &Order{}
+	for _, option := range options {
+		option(o)
+	}
+
+	return o
+}
+
+func (o *Order) GetColumn() string {
+	return o.column
+}
+
+func (o *Order) GetSort() Sort {
+	return o.sort
+}
+
+func (o *Order) SortString() string {
+	switch o.GetSort() {
+	case SortAscending:
+		return "ASC"
+	case SortDescending:
+		return "DESC"
+	}
+
+	return ``
+}
+
+func (o *Order) IsRaw() bool {
+	return o.isRaw
+}
+
+func ParseOrder(expressions ...string) []*Order {
```

Comment:
Please add more unit tests to test this method. And I think that putting this file in the orm directory is better.
The unit test should demo all cases. It's hard for me to understand how this method work.

And how about add `error` as return value? I think sometimes users may pass some invalid expressions.